myfirstcatapi/domains/cat_domain.py

```diff
@@ -10,7 +11,9 @@
 
 async def create_cat(new_cat: dto.UnsavedCat) -> dto.Cat:
     now = dates.get_utcnow()
-    return await cat_model.create_cat(new_cat, now=now)
+    cat = await cat_model.create_cat(new_cat, now=now)
+    fire_cat_created(cat.id)
```

Comment:
[suggestion] When adding code, also always ask yourself "How can I ensure this is working properly? How can I test this?"